zulipterminal/ui_tools/views.py

```diff
@@ -1498,6 +1500,7 @@ def keypress(self, size: urwid_Size, key: str) -> str:
                 and self.allow_update_emoji_list):
             self.allow_update_emoji_list = False
             self.emoji_search.set_edit_text('')
+            self.emoji_search.set_caption('Search [p]: ')
```

Comment:
This seems unnecessary :)

Reply:
That was actually necessary. This line resets the search caption to `Search [p]:`, after you press `Esc`(or move your focus from Search-box to Emoji list). If this line is left out, then the search caption would become empty after getting out of the search-box. However, if you have any other idea/refactor in mind then please let me know. :)

Reply:
I actually started comparing this search box with other search boxes; In other search boxes, the `Search [key]` indirectly acts as a sort of placeholder, which disappears when comes into focus; However, since the search box appears only on toggling the popup in this case, your re-setting the caption makes sense :+1: 

Also, we may also use f-strings and replace `p` via the `keys_for_command` to avoid that change in future :)